ownable-core/README.md

```diff
@@ -0,0 +1,34 @@
+[![crates.io](https://img.shields.io/crates/v/ownable-core.svg)](https://crates.io/crates/ownable-core)
+[![License](https://img.shields.io/crates/l/ownable-core.svg)](#license)
+
+# Core traits and types for the ownable crate
+
+<!--
+    update docs through `lib.rs` then sync here by running the following from
+    this crate's root
+    $ cargo rdme -w=ownable-core
+-->
```

Comment:
I opted for a personalized README, so the command to generate it needs to pass the specific workspace project since it goes against the default set in `.cargo-rdme.toml`